apps/cli/src/commands/bookmarks.ts

```diff
@@ -187,13 +187,19 @@ bookmarkCmd
   .description("update a bookmark")
   .option("--title <title>", "if set, the bookmark's title will be updated")
   .option("--note <note>", "if set, the bookmark's note will be updated")
+  .option("--stdin", "if set, the bookmark's note will be updated by stdin (not to be used with --note)")
```

Comment:
Can we give the flag a better name to indicate that it's updating the notes? Maybe `--note-stdin` or something?

Reply:
Indeed I wondered about `--stdin` for `add` parameter as well. It's unclear what it's for. But I decided to be consistent. I'ld suggest to rename `--stdin` for both commands after the merge? Or rename it in parallel and I'll update + adapbt before merging this PR?